Changelog.md

```diff
@@ -0,0 +1,71 @@
+# MSBuild 16.10.0
+
+This version of MSBuild will ship with Visual Studio 2019 version 16.10.0 and .NET SDK 5.0.300.
+
+## What's new
+
+* MSBuild now targets .NET 5.0 and .NET Framework 4.7.2.
+* MSBuild is faster and uses less memory.
+* Binary logs are smaller and have less performance overhead.
+* Tasks can now opt into resource management to improve parallelism in large builds.
+
+## Detailed release notes
+
+### Added
+
+* Projects can now specify `AdditionalTargetFrameworkInfoProperty` items to indicate that referencing projects should get those properties exposed as `AdditionalPropertiesFromProject` metadata on resolved reference items. (#5994).
+* The `Unzip` task now accepts `Include` and `Exclude` arguments to filter what is extracted from the zip file (#6018). Thanks, @IvanLieckens!
+* The `-graph:noBuild` command line argument can be used to validate that a graph is buildable without actually building it (#6016).
+* When building a solution filter, `$(SolutionFilterName)` is now defined (#6171).
+* `TaskParameterEventArgs` allow logging task parameters and values in a compact, structured way (#6155). Thanks, @KirillOsenkov!
+* ClickOnce publish now supports Ready To Run (#6244).
+* .NET 5.0 applications may now specify a toolset configuration file (#6220).
+* `ResolveAssemblyReferences` can now consume information about assemblies distributed as part of the SDK (#6017).
+* Allow constructing a `ProjectInstance` from a `ProjectLink` (#6262).
+* Introduce cross-process resource management for tasks (#5859).
```

Comment:
SolutionFilterName? Or still too up in the air? (I haven't seen a response on that, by the way.)

Reply:
https://github.com/dotnet/msbuild/pull/6351/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R19 ?